e2e-tests/insurance/cypress/e2e/journeys/policy/broker-manual-address/broker-manual-address-page.spec.js

```diff
@@ -0,0 +1,88 @@
+import { headingCaption } from '../../../../../../partials';
+import { brokerManualAddressPage } from '../../../../../../pages/insurance/policy';
+import { PAGES } from '../../../../../../content-strings';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../constants/field-ids/insurance/policy';
+import { POLICY_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/policy';
+import mockApplication from '../../../../../../fixtures/application';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.POLICY.BROKER_MANUAL_ADDRESS;
+
+const {
+  BROKER_DETAILS: { NAME, FULL_ADDRESS: FIELD_ID },
+} = POLICY_FIELD_IDS;
+
+const {
+  ROOT,
+  POLICY: { BROKER_MANUAL_ADDRESS_ROOT },
+} = INSURANCE_ROUTES;
+
+const { BROKER_MANUAL_ADDRESS: FIELD_STRINGS } = FIELDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Policy - Broker manual address page - As an exporter, ... TODO', () => {
```

Comment:
TODO comment?

Reply:
Thanks, we didn't have a fully populated ticket, we do now. Updated.